.github/workflows/build-userguide.yml

```
@@ -55,10 +57,12 @@ jobs:
         name: pdf-reference-guide
         path: ${{ env.PDF_PATH }}
 
-    - name: Upload user guide as release asset
+
+    - name: Publish assets
       if: ${{ env.IS_RELEASE == 'true' }}
-      uses: alexellis/upload-assets@0.4.0
       env:
-          GITHUB_TOKEN: ${{ github.token }}
-      with:
-          asset_paths: ${{ env.PDF_PATH_ASSET }}
+        GITHUB_TOKEN: ${{ github.token }}
+        tag: ${{ github.event.inputs.release_tag }}
+      run: |
+        gh release upload "$tag" ${{ env.PDF_PATH_ASSET }}      
```

See [logs](https://github.com/AntaresSimulatorTeam/Antares_Simulator/actions/runs/7478154922/job/20352486697)
```
 PDF_PATH: docs/pdf-doc-generation-with-sphinx/antares-general-reference-guide.pdf
 PDF_PATH_ASSET:  ["docs/pdf-doc-generation-with-sphinx/antares-general-reference-guide.pdf"]
 stat [docs/pdf-doc-generation-with-sphinx/antares-general-reference-guide.pdf]: no such file or directory
```
`PDF_PATH_ASSET` is an array with one element. I suggest using `PDF_PATH` instead.

```suggestion
        gh release upload "$tag" ${{ env.PDF_PATH }}      
```